src/components/App/AppLoader.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import LoadingScreen from '../LoadingScreen/LoadingScreen'
+
+const INTERNAL_APPS = ['home', 'permissions', 'apps', 'settings']
```

Comment:
I think we have a similar list somewhere else, perhaps in `MenuPanel`; it might be good to move them as direct exports from `static-apps`